hydra-cardano-api/src/Hydra/Cardano/Api/Value.hs

```diff
@@ -80,11 +78,6 @@ fromLedgerMultiAsset :: Ledger.MultiAsset StandardCrypto -> Value
 fromLedgerMultiAsset =
   fromMaryValue . Ledger.MaryValue 0
 
--- | Convert a cardano-api 'Value' into a cardano-ledger 'Value'.
-toLedgerValue :: Value -> Ledger.MaryValue StandardCrypto
-toLedgerValue =
-  toMaryValue
-
```

Comment:
IIRC we had added these "alias" functions to have consistent naming.

Should: not remove it to keep consistency.